### ISSUE TITLE
Auto3DSeg Learning Rate Scheduler Error when the num_interation is small (<5)

### ISSUE THREAD

**Issue by mingxin-zheng** [Project-MONAI/research-contributions#148]
**Describe the bug**

  File "./ref_api_work_dir/segresnet2d_0/scripts/train.py", line 279, in run
    lr_scheduler.step()
  File "/opt/conda/lib/python3.8/site-packages/torch/optim/lr_scheduler.py", line 161, in step
    values = self.get_lr()
  File "/opt/conda/lib/python3.8/site-packages/torch/optim/lr_scheduler.py", line 384, in get_lr
    if (self.last_epoch == 0) or (self.last_epoch % self.step_size != 0):

**To Reproduce**
Run the Auto3DSeg tutorials on a 8-GPU system

** Error Log**
